client/app/visualizations/edit-visualization-dialog.html

```diff
@@ -33,10 +33,18 @@ <h4 class="modal-title">Visualization Editor</h4>
       <div class="col-md-7" style="border: 1px solid #eee">
         <visualization-renderer visualization="$ctrl.visualization" query-result="$ctrl.queryResult"></visualization-renderer>
       </div>
+      <div class="col-md-7">  
+        <div class="bg-warning" ng-if="$ctrl.visualization.options.series.stacking == null && $ctrl.has3plusColumnsFunction()" style="padding:2%;" ng-bind-html="$ctrl.warning_three_column_stacking"> 
```

Comment:
Let's not do inline styles please. Also 2% may not work too well maybe something as a 'px' value.